Requires C++14

```
In file included from /usr/include/kj/memory.h:28,
                 from /usr/include/kj/array.h:28,
                 from /usr/include/kj/string.h:29,
                 from /usr/include/capnp/common.h:33,
                 from /usr/include/capnp/raw-schema.h:28,
                 from /usr/include/capnp/generated-header-support.h:30,
                 from /usr/include/capnp/schema.capnp.h:6,
                 from /usr/include/capnp/schema.h:32,
                 from /usr/include/capnp/dynamic.h:39,
                 from /home/vanyasem/Projects/AUR/_Unity8/_Unity8-Arch/unity-scopes-api-git/src/unity-scopes-api/include/unity/scopes/internal/zmq_middleware/ServantBase.h:21,
                 from /home/vanyasem/Projects/AUR/_Unity8/_Unity8-Arch/unity-scopes-api-git/src/unity-scopes-api/include/unity/scopes/internal/zmq_middleware/QueryCtrlI.h:23,
                 from /home/vanyasem/Projects/AUR/_Unity8/_Unity8-Arch/unity-scopes-api-git/src/unity-scopes-api/src/scopes/internal/zmq_middleware/QueryCtrlI.cpp:19:
/usr/include/kj/common.h:36:4: error: #error "This code requires C++14. Either your compiler does not support it or it is not enabled."
   #error "This code requires C++14. Either your compiler does not support it or it is not enabled."
    ^~~~~
/usr/include/kj/common.h:39:6: error: #error "Pass -std=c++14 on the compiler command line to enable C++14."
     #error "Pass -std=c++14 on the compiler command line to enable C++14."
      ^~~~~
```

```
In file included from /usr/include/kj/memory.h:28,
                 from /usr/include/kj/array.h:28,
                 from /usr/include/kj/string.h:29,
                 from /usr/include/capnp/common.h:33,
                 from /usr/include/capnp/raw-schema.h:28,
                 from /usr/include/capnp/generated-header-support.h:30,
                 from /home/vanyasem/Projects/AUR/_Unity8/_Unity8-Arch/unity-scopes-api-git/src/unity-scopes-api/build/src/scopes/internal/zmq_middleware/capnproto/Message.capnp.h:6,
                 from /home/vanyasem/Projects/AUR/_Unity8/_Unity8-Arch/unity-scopes-api-git/src/unity-scopes-api/include/unity/scopes/internal/zmq_middleware/ZmqObjectProxy.h:22,
                 from /home/vanyasem/Projects/AUR/_Unity8/_Unity8-Arch/unity-scopes-api-git/src/unity-scopes-api/include/unity/scopes/internal/zmq_middleware/ObjectAdapter.h:23,
                 from /home/vanyasem/Projects/AUR/_Unity8/_Unity8-Arch/unity-scopes-api-git/src/unity-scopes-api/src/scopes/internal/zmq_middleware/ObjectAdapter.cpp:19:
/usr/include/kj/common.h:36:4: error: #error "This code requires C++14. Either your compiler does not support it or it is not enabled."
   #error "This code requires C++14. Either your compiler does not support it or it is not enabled."
    ^~~~~
/usr/include/kj/common.h:39:6: error: #error "Pass -std=c++14 on the compiler command line to enable C++14."
     #error "Pass -std=c++14 on the compiler command line to enable C++14."
      ^~~~~
```

```
In file included from /usr/include/kj/memory.h:28,
                 from /usr/include/kj/array.h:28,
                 from /usr/include/kj/string.h:29,
                 from /usr/include/capnp/common.h:33,
                 from /usr/include/capnp/raw-schema.h:28,
                 from /usr/include/capnp/generated-header-support.h:30,
                 from /home/vanyasem/Projects/AUR/_Unity8/_Unity8-Arch/unity-scopes-api-git/src/unity-scopes-api/build/src/scopes/internal/zmq_middleware/capnproto/Message.capnp.h:6,
                 from /home/vanyasem/Projects/AUR/_Unity8/_Unity8-Arch/unity-scopes-api-git/src/unity-scopes-api/include/unity/scopes/internal/zmq_middleware/ZmqObjectProxy.h:22,
                 from /home/vanyasem/Projects/AUR/_Unity8/_Unity8-Arch/unity-scopes-api-git/src/unity-scopes-api/include/unity/scopes/internal/zmq_middleware/ObjectAdapter.h:23,
                 from /home/vanyasem/Projects/AUR/_Unity8/_Unity8-Arch/unity-scopes-api-git/src/unity-scopes-api/src/scopes/internal/zmq_middleware/Current.cpp:21:
/usr/include/kj/common.h:36:4: error: #error "This code requires C++14. Either your compiler does not support it or it is not enabled."
   #error "This code requires C++14. Either your compiler does not support it or it is not enabled."
    ^~~~~
/usr/include/kj/common.h:39:6: error: #error "Pass -std=c++14 on the compiler command line to enable C++14."
     #error "Pass -std=c++14 on the compiler command line to enable C++14."
      ^~~~~
```

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1,4 +1,6 @@
 cmake_minimum_required(VERSION 2.8.10)
+set(CMAKE_CXX_STANDARD 14)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
 
 # Default install location. Must be set here, before setting the project.
 if (NOT DEFINED CMAKE_INSTALL_PREFIX)

